contributing/standards/style-guide.md

```diff
@@ -51,7 +51,7 @@ An some examples of good usage:
 
 #### Oxford Comma
 
-Use the "[Oxford Comma](https://en.wikipedia.org/wiki/Serial_comma)" (also known as the Serial Comma) in lists. For example, rather than "I owe this to my parents, Ayn Rand and God", write "I owe this to my parents, Ayn Rand, and God". You may also enjoy [this image as an explanation](https://conchapman.files.wordpress.com/2015/12/panda2.jpg).
+Use the "[Oxford Comma](https://en.wikipedia.org/wiki/Serial_comma)" (also known as the Serial Comma) in lists. For example, rather than "I love my parents, Lady Gaga and Humpty Dumpty", write "I love my parents, Lady Gage, and Humpty Dumpty". You may also enjoy [this image as an explanation](https://conchapman.files.wordpress.com/2015/12/panda2.jpg).
```

Comment:
```suggestion
Use the "[Oxford Comma](https://en.wikipedia.org/wiki/Serial_comma)" (also known as the Serial Comma) in lists. For example, rather than "I love my parents, Lady Gaga and Humpty Dumpty", write "I love my parents, Lady Gaga, and Humpty Dumpty". You may also enjoy [this image as an explanation](https://conchapman.files.wordpress.com/2015/12/panda2.jpg).
```